src/tools/createMonitoringGoals.js

```diff
@@ -0,0 +1,153 @@
+import {
+  sequelize,
+  GoalTemplate,
+} from '../models';
+import { auditLogger } from '../logger';
+
+const createMonitoringGoals = async () => {
+  const cutOffDate = '2024-10-01'; // TODO: Set this before we deploy to prod.
```

Comment:
Set this to what before we deploy to prod?

Reply:
It will be the day we deploy.